### ISSUE TITLE
Mounting filesystem...

### ISSUE THREAD

**Issue by grkka** [Samgisaninja/SuccessionRestore#37]
When I click on the message: "Mounting filesystem ... Tap OK to continue" nothing happens.
iPhone Xs Max, iOS 12.1.2, Succession 1.3.3

**Comment by FounderHawk** [comment]
What version are you using? I should add: How long ago did you install succession, as 1.3.3 was just released within 30 minutes

**Comment by Samgisaninja** [comment]
Are you sure you're on 1.3.3.... I only published 1.3.3 23 minutes before this issue was created, and it has not yet been released on the default repos

**Comment by grkka** [comment]
Yes, I downloaded the .deb file from GitHub XD

**Comment by Samgisaninja** [comment]
ok, can you please turn on the "log output" switch in settings, cause the issue, then upload `/var/mobile/succession.log`?

**Comment by grkka** [comment]
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3243834/succession.log)

**Comment by Samgisaninja** [comment]
ok, now can you use mterminal/newterm2/ssh, type `ls /var/MobileSoftwareUpdate/mnt1/sbin/`

**Comment by Samgisaninja** [comment]
send output here

**Comment by grkka** [comment]
![Captura de pantalla 2019-06-01 a las 16 26 31](https://user-images.githubusercontent.com/14794477/58749745-0b36ba00-848a-11e9-81dc-de97add522be.png)

**Comment by Samgisaninja** [comment]
hmm, can you now run `ls /dev`

**Comment by grkka** [comment]
![Captura de pantalla 2019-06-01 a las 16 32 44](https://user-images.githubusercontent.com/14794477/58749811-e858d580-848a-11e9-853b-6980a81677a4.png)

**Comment by Samgisaninja** [comment]
ok, type `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1`

I'll leave this open to see if I get any more reports of this

# EDIT: DO NOT take that command out of context and use it on your device and expect it to work. I wrote this comment a while ago and I've seen it reposted everywhere since, with people mad at me because it didn't work for you. \*\*That's because I wasn't talking to you\*\*

**Comment by Samgisaninja** [comment]
(after you type that, succession will work)

**Comment by grkka** [comment]
![IMG_3458](https://user-images.githubusercontent.com/14794477/58749874-96647f80-848b-11e9-85d1-2496b5aa1c2b.jpeg)
Done!

**Comment by grkka** [comment]
Process finished successfully. Restoration completed. Thank you!

**Comment by putrajay** [comment]
Me too “Mounting filesystem ... Tap OK to continue" nothing happens.
![8253EFE9-9265-4BB4-9A12-64B0E4058DB3](https://user-images.githubusercontent.com/51270146/58751371-c29df180-84c7-11e9-8358-620eebdfd9db.png)

**Comment by putrajay** [comment]
![56EB0AD1-EB16-4676-A82A-4B5EC5BADA9F](https://user-images.githubusercontent.com/51270146/58751516-a00cd800-84c9-11e9-8cfc-1e4db71a8fda.jpeg)

**Comment by TheRealKeto** [comment]
> ok, type `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1`
> 
> I'll leave this open to see if I get any more reports of this

This command must be run as root in MTerminal/NewTerm2. Simply type in `su`, followed by your password (default is `alpine`) and then run the command. 

After, simply go back to Succession and the restore should work just fine.

**Comment by onlyboda** [comment]
I chose the option for full restore after restore is done jailbreak icons still there in springboard why ?

**Comment by grkka** [comment]
It happened to me too. Apps like Filza, iCleaner or Sileo would have to be deleted via SSH before using Succession

**Comment by onlyboda** [comment]
How to do that pls?

**Comment by grkka** [comment]
Connecting to the iPhone via SSH, I suppose that the Filza, Sileo and iCleaner apps will have to be eliminated in the / var / Aplicattions route.  And then restore with Succession

**Comment by TomKFL** [comment]
![IMG_2843](https://user-images.githubusercontent.com/51007649/58768866-70c99a00-85a1-11e9-9b60-6377751983ed.PNG)
![IMG_2845](https://user-images.githubusercontent.com/51007649/58768867-78893e80-85a1-11e9-9e6c-c2be02bd2828.PNG)
![IMG_2846](https://user-images.githubusercontent.com/51007649/58768869-7aeb9880-85a1-11e9-81da-06b5195f439a.PNG)
![IMG_2847](https://user-images.githubusercontent.com/51007649/58768899-d3bb3100-85a1-11e9-8c44-6bd10a9ba69e.PNG)
Wont work for me. A12 iPhone 11,6 iOS 12.0 16A366

**Comment by TheRealKeto** [comment]
> It happened to me too. Apps like Filza, iCleaner or Sileo would have to be deleted via SSH before using Succession

@grkka I originally thought that icons and such were also going to be deleted, however, upon jailbreaking with u0 again, I realized they were not deleted. This is probably due to how the DMG was mounted by the command that Samg left. Not sure why Succession itself can't mount it the way it's supposed to, but for now, my guess is that this is just a workaround to Succession to work. Perhaps Samg could provide more info on how restoring the file system and how mounting the DMG that way affects the restoring process.

**Comment by Samgisaninja** [comment]
Alright so I just had a chance to see this, first off thanks everyone for your patience and the logs

In everyone here, the first indication that anything has gone wrong is when line 146 (theDiskString) logs. 

The first issue, being experienced by @tomkfl, I know what your issue is, for some reason, attach is returning the actual filesystem's disk, not where the dmg was attached. I don't know *why* that's happening, but I know how to fix it, and I will shortly, probably tomorrow or the next day, depending on when I get a change

In @putrajay 's case, attach doesn't appear to be returning a char array for theDisk. Can you (putrajay) please send me your device model and iOS version, also try downgrading to version 1.3.2 and tell me if it works? (1.3.3 implemented a new way of mounting the attached disk image, but I'd attach doesn't return a theDisk... it doesn't work, but the old method might still work, if it does I can just use that as a fallback, so please try that and report back)

Then there's the third kind, @grrka 's. His log shows theDisk being correctly identified, AND it shows that the mount succeeded (even though I told him to mount disk4, it is clearly also attached to disk6, as proven by the screenshots). Maybe I should get Succession to always prefer disk4s2s1, even if attach returns something different? This raises the possibility that someone could've mounted another DMG on their device without rebooting before using succession, but I don't believe there are any other publicly available tools that deal with mounting (part of the reason this is so difficult--I have no one to learn from for how to do this)

**Comment by Samgisaninja** [comment]
edit: @grkka

**Comment by BirdoGuy** [comment]
I’m having the same issue as well. It says “ERROR! unable to identify theDisk, neither s2s1 or s2 existed”

**Comment by BirdoGuy** [comment]
I’m having the same issue as well. It says “ERROR! unable to identify theDisk

**Comment by Samgisaninja** [comment]
Can you please send the whole log? @birdoguy

**Comment by grkka** [comment]
Thanks, I do not understand the explanation very well because I do not know much about computer science. What can you explain, it could be that at the beginning Succession did not work and I downloaded the ipsw file and added it with Filza (later I deleted it) and then I downloaded it through Succesion?
"This raises the possibility that someone could've mounted another DMG on their device without rebooting before using succession"

**Comment by alitez** [comment]
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3247111/succession.log)
hi. i have a different error here. i have 5s iPhone6,2 ios 11.3.1 unc0ver 3.2.1 .
Edit: i deleted MobileSoftwareUpdate folder. now the last line of the log is: 
Line 506: ERROR! unable to identify theDisk, neither s2s1 or s2 existed
Edit2: Succession 1.0 (0.9 beta6) works for me now. But i am having issues with daemons(idk is this related to succession.)

**Comment by zzaibis** [comment]
I have the same issue as well

**Comment by brcwayne** [comment]
hi
i am on 6s+ ios 12.1.1 and 
succession 1.3.3 
uncover 3.1.3
and i cleaned up all tweaks

i write down that code to new term2 and succession worked. it shows the numbers and reboot the phone than everything remain same. nothing happens but the jailbreak. 
unc0ver works fine again.
i wanted to try again succession give me same error i mean dosent work so i retype the code again and jump back to succession and press erase but tweak crashes. Nothing work again. so i just give up and waiting your update. 

Thank you for your effort.
by the way i cant take the log sorry.

**Comment by 4rtz1z** [comment]
Hi Guys,
I have the same issues, and i think my orig-fs now kinda messed up (the App and the tweaks is not gone),
i'm on iphone X, 12.1.1, Uncover 3.2.1.

1. The mounting is the problem (i follow @Samgisaninja command line), yes it works
2. After succeeded and rebooting, the JB Files is still there, like Cydia, Filza, Zebra, Snapback etc.
3. I tried the create orig-fs but after a reboot, it is still the same like point 2
4. I tried restore system only and create orig-fs, but it get stuck at rebuilding icon cache and succession goes black, and nothing happened, then i reboot the device, and nothing really happened like point 2.

i've attached my log, hopefully it could help figure something out.
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3255917/succession.log)

Edit :
Finally succeed after downgrade UIKit Tools to 1.1.13-4 and using Succession 1.3

1. The mounting worked
2. Restoring FS worked
3. Now orig-fs is clean

**Comment by rafpodgorski** [comment]
Same problem on iphone SE, ios 12.1.1, unc0ver 3.2.1

**Comment by huynhthanhloc** [comment]
> ok, type `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1`
> 
> I'll leave this open to see if I get any more reports of this

![2019_06_06_00_31_IMG_0001](https://user-images.githubusercontent.com/9433332/58976989-f09a7300-87f2-11e9-877b-1fbdc94ed083.PNG)
i don't see disk4s2s1. so how must i do ?

**Comment by Samgisaninja** [comment]
@abdulazizalfauzi your issue has nothing to do with APFS snapshots.

After you got it to mount, it appears you had a completely different (and unrelated) problem. To make this easier for me to track, can you please create a new issue on this github and re-attach that log file? Thanks.

**Comment by Samgisaninja** [comment]
@huynhthanhloc mount disk4s2 instead

**Comment by huynhthanhloc** [comment]
> @huynhthanhloc mount disk4s2 instead

![2019_06_06_03_59_IMG_0002](https://user-images.githubusercontent.com/9433332/58990219-a2479d00-880f-11e9-97f1-dd51adf802a6.PNG)
it get error .how to fix it bro? 
( i use 6s 10.2)

**Comment by Samgisaninja** [comment]
lol, if you're on 10.2 you obviously can't mount an APFS type filesystem... APFS wasn't introduced until 10.3... so use HFS???

**Comment by huynhthanhloc** [comment]
> lol, if you're on 10.2 you obviously can't mount an APFS type filesystem... APFS wasn't introduced until 10.3... so use HFS???

i don't know it, now i replace APFS to HFS ,and . ... 
![2019_06_06_07_09_IMG_0003](https://user-images.githubusercontent.com/9433332/58998270-2d359100-882a-11e9-996f-c63e7816cb9e.PNG)

**Comment by huynhthanhloc** [comment]
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3259618/succession.log)
![2019_06_06_07_44_IMG_0004](https://user-images.githubusercontent.com/9433332/58999364-a8e60c80-882f-11e9-9247-fa61a3352940.PNG)

it ran, and stop here.

**Comment by liu-congcong** [comment]
![image](https://user-images.githubusercontent.com/34596618/59102694-6de2f680-895f-11e9-9a04-d5a1d993bc62.png)
I mount disk3s2s1 (not always disk4s2s1) to mnt1 and restore successfully :)

**Comment by Samgisaninja** [comment]
@huynhthanhloc ...you had "test mode" in settings turned on.

**Comment by ar1s3** [comment]
Hi all,
after some test opening succession.log i have this error:
 ERROR! You don’t have permission to save the file “mnt1” in the folder “MobileSoftwareUpdate”
Someone could help me with this?
iOS 12.0.1 - iPhone 6S

Regards

**Comment by aulianad** [comment]
Can help me? Please sir
![EC04C3C1-3AB5-4B8A-9158-8C2E79E6B6C5](https://user-images.githubusercontent.com/51692597/59247009-3b2f4b80-8c48-11e9-87ed-724a744caaa1.png)
![545F09DF-6EDE-4FD0-AB44-FDF18C20F6A5](https://user-images.githubusercontent.com/51692597/59247010-3bc7e200-8c48-11e9-81f7-f061804a869c.png)
![E06FA8F8-168C-40E4-B7BB-3BC5DC6EC5B7](https://user-images.githubusercontent.com/51692597/59247011-3bc7e200-8c48-11e9-8159-699fde221dbc.png)

**Comment by joe136685182** [comment]
I had the same problem, then I ran `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1` and the process continued.
But when the process went to "Restore in progress" and showing like "13300 files", the app got frozen and the device rebooted a minute later.
I have to re-jailbreak my iPhone, re-run the mount command and try again. The file count was different in different freeze, and the log file is as follow.
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3275277/succession.log)

**Comment by aulianad** [comment]
thx workk.
![C30F077A-037F-43B7-BC44-A65CE3AE1049](https://user-images.githubusercontent.com/51692597/59262264-15666e80-8c69-11e9-86d7-5803e8bc4587.jpeg)

**Comment by LaxusiC** [comment]
How do I unmount after using the command above @Samgisaninja

**Comment by Samgisaninja** [comment]
The latest commit may have fixed this, please install the following deb: https://raw.githubusercontent.com/Samgisaninja/test/master/debs/com.samgisaninja.successionrestore_1.3.4-b2_iphoneos-arm.deb


If anyone still has this issue, please send me a log

**Comment by igobythisname** [comment]
THANK YOU grkka!!! and THANK YOU Samg!!!

I was also getting stuck with "Mounting filesystem Tap OK" with my 6S Plus on iOS 11.1.2...

took me a hot min to resolve -since MTerminal doesn't work on iOS 11 w/the default repos.. had to spend time researching, discovered MTerminal has been tweaked to work for iOS 11, need to get it from Elucubratus Repo; once I got that working, the command line fix Samg gave here worked like a charm! my 6S just reached the Apple logo screen, yesssss! Thank you!!

Samg I hope you got my donation :D

edit: okay, not so much successful. after completing Setup as new iPhone after running Succession, Cydia, Filza, Sileo, icons are still present, opening up Cydia gives a bunch of errors (in yellow, filesystem related I think); I can't install any packages or anything. I've tried re-jailbreaking w/Electra, but it doesn't help. I'm trying to use Succession 1.3.4-b2 posted above but Sileo isn't working and Filza is telling me unable to access dpkg status area: no such file or directory bash-4.4$. I'm trying to run Succession again, got 'Mounting filesystem...' pop-up again, although idk how I'm going to use MTerminal this time.. :-/

**Comment by ghost** [comment]
> ok, type `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1`
> 
> I'll leave this open to see if I get any more reports of this

This worked for me with the latest Succession on iPhone 7 GSM.

**Comment by ghost** [comment]
Hi sir can you tell me
before I can’t mounting filesystem and then I use your cmd code mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1 is worked but when I setup comple cydia and all of my tweak is doesn’t delete what happen ?@Samgisaninja

**Comment by Samgisaninja** [comment]
That actually sounds a lot like #44, can you please send a log to *that issue*

**Comment by ghost** [comment]
So I thought `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1` fixed the issue for me on iPhone 7 GSM, but just like KheangRio, nothing was restored after completion. The Succession icon was still present too. No services were disabled #44. It was a fresh jailbreak with no changes made.

As restoring my 200GB iTunes backup takes ~4 hours and I rely on this iPhone, I decided to settle with just a futurerestore in the end. Sorry I cannot provide more valuable info.

**Comment by ndwhyy** [comment]
Hey man, I also have the issue when I get  "Mounting filesystem..." and I  tap "OK to continue" and nothing happens. I'm on iOS 12.2, iPhone 8 & latest unc0ver. Hopefully this info will help you to troubleshoot the issue.

Here's the log:
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3407315/succession.log)

Here's the output of ls `/var/MobileSoftwareUpdate/mnt1/sbin/` and `ls /dev`
[commands.txt](https://github.com/Samgisaninja/SuccessionRestore/files/3407319/commands.txt)

**Comment by shadowscott** [comment]
Have he a same issue on my 5s on 12.1.1 lastest succession. 
https://pastebin.com/P6gpTk5U

**Comment by SatoroX** [comment]
> ok, type `mount -t apfs -o ro /dev/disk4s2s1 /var/MobileSoftwareUpdate/mnt1`

![4BD6EFB9-EB9E-4C16-B8CE-B451C0FAE6F8](https://user-images.githubusercontent.com/53506793/62187378-dbb10a80-b371-11e9-96e6-499af25919ba.png)

So i don’t have disk4 or disk3 
I'm on ios 12.2

**Comment by SatoroX** [comment]
![603570C2-8AF7-48D2-87FA-07B132AB037C](https://user-images.githubusercontent.com/53506793/62193179-e6729c00-b37f-11e9-8771-5bd66c6ddda6.png)
here is the log

**Comment by SatoroX** [comment]
![713274D4-0772-4441-BAA7-04A974A3A4CA](https://user-images.githubusercontent.com/53506793/62194222-1c188480-b382-11e9-9225-ebd479036fb6.png)

this is the log

**Comment by ilya2013-debug** [comment]
Help me , I can’t erase my iPad cuz mounting filesystem!

Here is my log.
![Image](https://i.imgur.com/LvJEec7.jpg)

I am on iPad Air 2 12.1.4

**Comment by cyntrifical** [comment]
Running iOS 11.4.1 on iPhone 6, with unc0ver 3.4.2, using Succession v. 1.3.4, and latest version of Bingners Snappy (2.1?) anyhow same issue for me as well. I’ve downloaded the proper IPSW through succession, selected to not save user data, and to use the option to overwrite the APFS snapshot with a new orig-fs, and upon clicking “Prepare for restore!”, “Erase iPhone”, I’m met with the “Mounting filesystem... Tap OK to continue.” And of course after selecting “OK” nothing happens. I’ve logged onto succession.log within /var/mobile/succession.log, please let me know what further steps to take from here as I am attempting to clear this iPhone, while retaining iOS 11.4.1, so as to sell it with the possibility of being Jailbreakable to the next buyer. Thank you so much in advance Samg!

[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3464688/succession.log)
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3464689/succession.log)

**Comment by grkka** [comment]
Hello, I have used Succession again on an iPhone XS Max iOS 12.1.2 and everything has worked at first and perfectly.  Succession has removed apps like Filza and iCleaner. All perfect. Thank you.

**Comment by Samgisaninja** [comment]
I have made another attempt to fix this issue in 1.3.5... but again I'm still not sure what the cause was...

I've tested it multiple times and it never happened, but same with all the other versions ¯\\\_(ツ)_/¯

1.3.5 is available on my repo now and should be on the public repos in the next 24-48 hours

**Comment by cyntrifical** [comment]
Thanks for sending!   
  
  
  

  
  

  
  
>

**Comment by Samgisaninja** [comment]
alright, yay, so the mounting appears to finally be fixed.

unfortunately, attaching is now broken, which causes the same behavior. This is shown in the log when hdik fails, which will log something like this:

`Line 218: hdik output is: ` (line number might be different)

Since this issue is completely unrelated to mounting failing, I am closing this issue report. If you experience the "new version" of this issue where attaching fails because hdik outputs nothing, please discuss it in #48

**Comment by gladsonloureto** [comment]
How to resolve this error?

Line 636: ERROR! Attaching image failed because hdik returned no output. Please reboot and try again.

**Comment by wlh16** [comment]
i have the same issue on my iphone xs max succession 1.3.7 ios 12.1 plz i need help i cant lose my jaillbreak 
![WhatsApp Image 2019-08-23 at 02 32 34](https://user-images.githubusercontent.com/5621244/63560604-8ae69900-c54e-11e9-9754-8b700d1a5563.jpeg)
![WhatsApp Image 2019-08-23 at 02 32 34 (1)](https://user-images.githubusercontent.com/5621244/63560607-8c17c600-c54e-11e9-9b6c-1e8a0f50d4c5.jpeg)

**Comment by BiasShadow** [comment]
![386506C0-5D13-4523-AF1E-CB0817061613](https://user-images.githubusercontent.com/45723473/65870691-c3805a80-e3b7-11e9-9e93-dd63747fd965.png)
![A2E6017D-0AD1-4D09-BCFB-0EBE55CCB3F3](https://user-images.githubusercontent.com/45723473/65870627-97fd7000-e3b7-11e9-9a99-88fbed43ef90.png)
![F3CB9842-1A4B-44D3-BA93-AEA4BD7F4066](https://user-images.githubusercontent.com/45723473/65870628-97fd7000-e3b7-11e9-99c5-e0552808cec4.png)
Having this issue on the latest version of Succession, just not the latest Uncover. I read up above that some with similar issues mentioned Filza and other certain tweaks. Does this affect Succession from functioning properly? I also have only restore system data option toggled on. All other options are default.

**Comment by mclopes** [comment]
Latest version of Succession (updated today) 
Iphone x
12.4

![IMG_3432](https://user-images.githubusercontent.com/5324084/65979187-dbe98700-e442-11e9-9d8c-deaf3926649a.jpg)
![IMG_3431](https://user-images.githubusercontent.com/5324084/65979186-dbe98700-e442-11e9-8514-f19d98a2124c.jpg)

**Comment by akeebfahiz** [comment]
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/4128791/succession.log)
any solution

**Comment by akeebfahiz** [comment]
> ok, can you please turn on the "log output" switch in settings, cause the issue, then upload `/var/mobile/succession.log`?

[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/4128826/succession.log)

**Comment by oz6ur** [comment]
hello. help please waiting on this screen. XS Max
''Attaching This Should Take Less Than 10 Seconds''


[SUCCESSIONLOG 1.4.11: 2020-02-21 17:48:47 +0000] Line 92: Mountpoint exists, continuing
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:48:47 +0000] Line 167: attachDiskImage called!
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:48:47 +0000] Line 170: using hdik to attach disk image
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:48:47 +0000] Line 175: hdik /private/var/mobile/Media/Succession/rfs.dmg
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:48:47 +0000] Line 183: hdik completed with
hdik: must be run by root

[SUCCESSIONLOG 1.4.11: 2020-02-21 17:48:47 +0000] Line 214: changedDevContents: 
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:13 +0000] Line 92: Mountpoint exists, continuing
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:13 +0000] Line 167: attachDiskImage called!
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:13 +0000] Line 170: using hdik to attach disk image
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:13 +0000] Line 175: hdik /private/var/mobile/Media/Succession/rfs.dmg
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:13 +0000] Line 183: hdik completed with
hdik: must be run by root

[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:13 +0000] Line 214: changedDevContents: 
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:36 +0000] Line 92: Mountpoint exists, continuing
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:36 +0000] Line 167: attachDiskImage called!
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:36 +0000] Line 170: using hdik to attach disk image
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:36 +0000] Line 175: hdik /private/var/mobile/Media/Succession/rfs.dmg
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:36 +0000] Line 183: hdik completed with
hdik: must be run by root

[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:36 +0000] Line 214: changedDevContents: 
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:46 +0000] Line 92: Mountpoint exists, continuing
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:46 +0000] Line 167: attachDiskImage called!
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:46 +0000] Line 170: using hdik to attach disk image
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:46 +0000] Line 175: hdik /private/var/mobile/Media/Succession/rfs.dmg
[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:46 +0000] Line 183: hdik completed with
hdik: must be run by root

[SUCCESSIONLOG 1.4.11: 2020-02-21 17:52:46 +0000] Line 214: changedDevContents:

**Comment by Samgisaninja** [comment]
^ duplicate of #89

**Comment by iFhone4s** [comment]
Hi, Thx for support!
I try use succession on iphone 4s ios 9.3.2, decrypted ispw and all ok, but when press erase: Can´t restore, filesystem isn´t mounted.
Succession.log :
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 92: Mountpoint exists, continuing
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 167: attachDiskImage called!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 262: Using comex attach for attach
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 275: attach output is: disk2

[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 278: disk2, 

2
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 296: found attached diskname disk2
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 303: sending /dev/disk2 to mountRestoreDisk
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 338: prepareMountAttachedDisk called!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 350: Identified filesystem as HFS!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:31 +0000] Line 375: mounting complete! 
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:33 +0000] Line 392: tappedRestoreButton called
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:33 +0000] Line 435: no apfs snapshot operations requested
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:33 +0000] Line 441: showRestoreAlert called!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:19:33 +0000] Line 502: ERROR! Can't restore, filesystem isn't mounted.
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:57 +0000] Line 92: Mountpoint exists, continuing
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:57 +0000] Line 167: attachDiskImage called!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:57 +0000] Line 262: Using comex attach for attach
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 275: attach output is: disk3

[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 278: disk3, 

2
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 296: found attached diskname disk3
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 303: sending /dev/disk3 to mountRestoreDisk
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 338: prepareMountAttachedDisk called!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 350: Identified filesystem as HFS!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:47:58 +0000] Line 375: mounting complete! 
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:48:00 +0000] Line 392: tappedRestoreButton called
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:48:00 +0000] Line 435: no apfs snapshot operations requested
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:48:00 +0000] Line 441: showRestoreAlert called!
[SUCCESSIONLOG 1.4.11: 2020-03-02 11:48:00 +0000] Line 502: ERROR! Can't restore, filesystem isn't mounted.

Help please!
